app/addons/documents/routes-index-editor.js

```diff
@@ -57,6 +58,10 @@ var IndexEditorAndResults = BaseRoute.extend({
     this.createDesignDocsCollection();
     this.addLeftHeader();
     this.addSidebar();
+
+    this.setComponent("#right-header", RightAllDocsHeader, {
+      database: this.database
+    });
```

Comment:
just one page gets the alldocsheader as far as i know, now all three routes get it?